.github/workflows/buildsCI.yaml

```diff
@@ -400,4 +400,34 @@ jobs:
        cmake --build . --config ${{env.LRS_BUILD_CONFIG}} -- -j4
        ls
   
-  
+  cppcheck:
+    name: cppcheck-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+          
+      - name: cppcheck_run
+        uses: deep5050/cppcheck-action@main
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN}}
```

Comment:
What does it means?

Reply:
Supposed to get your permissions to the repository to run the public action on your behalf. I removed and am using my own bash script instead of the public action